metadata.yaml

```diff
@@ -6,4 +6,17 @@ description: |
   As the next-generation Logstash Forwarder, Filebeat tails logs and quickly
   sends this information to Logstash for further parsing and enrichment or to
   Elasticsearch for centralized storage and analysis.
-series: []
+series: [bionic, focal, jammy]
+subordinate: true
+tags:
+  - filebeat
+requires:
+  beats-host:
+    interface: juju-info
+    scope: container
+  logstash:
+    interface: elastic-beats
+  elasticsearch:
+    interface: elasticsearch
+  kafka:
+    interface: kafka
```

Comment:
hey @esunar and @agileshaw I've pushed some changes which I hope will help with the issues seen. Seems that build no longer works on my workstation, I had to add these lines to have a successful build.

Reply:
The changes looks good to me. Could you please attach the log from some simple bundle deployment? (e.g. https://git.launchpad.net/charm-graylog/tree/src/tests/functional/tests/bundles/base-graylog.yaml). I would like to see all if this jammy-built charm doesn't break such a deployment.

Reply:
hey sure, this is the bundle I am using today:

https://pastebin.canonical.com/p/kcddqf7WsQ/

Reply:
And do you have a `juju status` (or something similar that shows deployment status) output of the deployed bundle?

Reply:
sure, juju status is here: https://pastebin.canonical.com/p/jFRJTx3x7p/